extension/src/popup/components/accountMigration/ReviewMigration/index.tsx

```diff
@@ -172,46 +267,52 @@ export const ReviewMigration = () => {
             (2 + account.subentry_count) * BASE_RESERVE,
           ).toString();
 
+          const xlmBalance =
+            account.balances[account.balances.length - 1].balance;
+          const dataEntries = Object.keys(account.data_attr).length;
+          const trustlineBalances = account.balances.filter(
+            ({ asset_type: assetType }) => assetType !== "native",
+          );
+
           acctItem = {
             ...DEFAULT_ACCT_ITEM,
             trustlines: account.balances.length - 1,
-            dataEntries: Object.keys(account.data_attr).length,
-            xlmBalance: account.balances[account.balances.length - 1].balance,
-            trustlineBalances: account.balances.filter(
-              ({ asset_type: assetType }) => assetType !== "native",
-            ),
+            dataEntries,
+            xlmBalance,
+            trustlineBalances,
             minBalance,
           };
+        } else {
+          hasUnmigratableAccount = true;
```

Comment:
similarly, if an account is not found (and therefore unfunded), block submission